src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/CryptoConfig.cs

```diff
@@ -15,6 +15,7 @@ public class CryptoConfig
         private const string AssemblyName_Csp = "System.Security.Cryptography.Csp";
         private const string AssemblyName_Pkcs = "System.Security.Cryptography.Pkcs";
         private const string AssemblyName_X509Certificates = "System.Security.Cryptography.X509Certificates";
```

Comment:
Yes, but also you will need to checkout CryptoConfig to dotnet/corefx/master after it is merged